image-spec.csv

```diff
@@ -5,4 +5,5 @@ context_dir,image_tag,from_tag
 ./kube-tools/latest,kube-tools,
 ./node-setup,node-setup,base-ubi-9.4-minimal
 ./poetry/1.5,poetry-1.5,
```

Comment:
why are we still building 1.5?

Reply:
I'll send a separate PR to remove it, I didn't want to do it before all the jobs started passing with 1.8

Reply:
ok, thx - btw. the old 1.5 image would still be there, it would just not receive updates, so this may not need a 2 step process next time.